src/Header/Account/Account.jsx

```diff
@@ -167,7 +196,11 @@ Account.propTypes = {
 		links: PropTypes.object
 	}),
 	environment: PropTypes.oneOf(["live", "test", "beta", "local"]),
-	provider: PropTypes.oneOf(["niceAccounts"])
+	provider: PropTypes.oneOf(["niceAccounts", Account.idamProvider]),
+	links: PropTypes.arrayOf(
+		PropTypes.shape({ name: PropTypes.string, value: PropTypes.string })
```

Comment:
Isn't it key not name? Also, key and value are both required within the shape:
```suggestion
		PropTypes.shape({ key: PropTypes.string.isRequired , value: PropTypes.string.isRequired  })
```